drivers/mspi/mspi_nrfe.c

```diff
@@ -254,27 +260,35 @@ static int nrfe_mspi_wait_for_response(nrfe_mspi_opcode_t opcode, uint32_t timeo
 }
 
 /**
- * @brief Send a configuration struct to the FLPR core using the IPC service.
+ * @brief Send data to the FLPR core using the IPC service, and wait for FLPR response.
  *
  * @param opcode The configuration packet opcode to send.
- * @param config The data to send.
+ * @param data The data to send.
  * @param len The length of the data to send.
  *
  * @return 0 on success, negative errno code on failure.
  */
-static int send_config(nrfe_mspi_opcode_t opcode, const void *config, size_t len)
+static int send_data(nrfe_mspi_opcode_t opcode, const void *data, size_t len)
 {
 	int rc;
-	rc = mspi_ipc_data_send(opcode, config, len);
+
+#ifdef CONFIG_MSPI_NRFE_IPC_NO_COPY
+	(void)len;
+	void *data_ptr = (void *)data;
+
+	rc = mspi_ipc_data_send(opcode, &data_ptr, sizeof(void *));
```

Comment:
The fact that the length is not used in this case is a little confusing, but I understand that you do not want to modify lines where `send_data` is called.